docs/metrics/getting-started-async-counter/README.md

```diff
@@ -22,7 +22,7 @@ Install the
 package:
 
 ```sh
-dotnet add package OpenTelemetry.Exporter.Console
+dotnet add package OpenTelemetry.Exporter.Console --version 1.2.*.*
```

Comment:
@mic-max Please check the above. I think @utpilla is right.

Reply:
Yes, you're right @utpilla I meant to put `--version 1.2.*-*` with the hyphen instead of period.
Your suggestion is also better so I will update to use that 👍